picoshare/picoshare.go

```diff
@@ -25,6 +25,9 @@ type (
 		Size          FileSize
 		GuestLink     GuestLink
 		DownloadCount uint64
+		// PassphraseKey stores a serialized kdf.DerivedKey if the uploader
+		// protected the file with a passphrase. Empty means no protection.
+		PassphraseKey string
```

Comment:
This should be a `kdf.DerivedKey` and not a `string`. The `sqlite` and `handlers` packages are responsible for serializing and deserializing it from string to the correct type.